server/views/pages/prison-register/allPrisons.njk

```diff
@@ -68,6 +70,11 @@
                                                     Closed
                                                 {% endif %}
                                             </td>
+                                            <td class="govuk-table__cell">
+                                                {% for type in prison.types %}
+                                                    {{ type.code }}{% if loop.revindex > 1 %},{% endif %}
```

Comment:
This isn't the easiest to read, however having it as the following
```
{{ type.code }}
{% if loop.revindex > 1 %}
  ,
{% endif %}
```
Meant the types would display with a space before the comma, for example `HML , YOI`

Inline ensured there was no space before the comma, for example `HML, YOI`

Not a huge one but thought a neater UI took precedence here, happy to change it

Reply:
yeah - does look a tad messy...Wondered if we could anything with the "join"... ha ha - can do it nicely in kotlin!  Don't think this is a blocker tho